config/options

```diff
@@ -4,77 +4,73 @@ if [[ $EUID -eq 0 ]]; then
 fi
 
 # set default language for buildsystem
-  export LC_ALL=C
+export LC_ALL=C
 
 # set default independent variables
 ROOT="${PWD}"
-DISTRO_DIR="$ROOT/distributions"
-PROJECT_DIR="$ROOT/projects"
+DISTRO_DIR="${ROOT}/distributions"
+PROJECT_DIR="${ROOT}/projects"
 
 # determines DISTRO, if not forced by user
 # default is LibreELEC
-if [ -z "$DISTRO" ]; then
-  DISTRO="LibreELEC"
-else
-  DISTRO="$DISTRO"
-fi
+DISTRO="${DISTRO:-LibreELEC}"
```

Comment:
The comment `# default is LibreELEC` (or Generic or x86_64) is probably superfluous? If it must be stated, I'd tack it on to the the end of the first comment.